docs/developers/v2/hacking-with-yearn.md

```diff
@@ -2,145 +2,153 @@
 
 This page is for the hackers and buidlers who are currently building, or planning to build, something cool with Yearn Finance. This guide details all the info you need to start hacking!
 
-# Core Tools
+## Core Tools
 
 - [Solidity](https://docs.soliditylang.org/en/v0.8.6/index.html)
```

Comment:
this is an old version of solidity. Going here allows access to the newest version. Is there a reason that it links to v0.8.6?
New version is https://docs.soliditylang.org/en/v0.8.23/

Reply:
I checked a v2 strategy (lido weth) and it uses v0.6.12, vaults are in vyper, so I don't see a reason for a specific version here u can maybe just put the latest!

Reply:
done. because there is no "latest" page, this may need to be manually updated in the future as versions change.